cmd/internal/cli/sign.go

```diff
@@ -131,13 +132,17 @@ func doSignCmd(cmd *cobra.Command, cpath string) {
 	// Set key material.
 	switch {
 	case cmd.Flag(signPrivateKeyFlag.Name).Changed:
+		fmt.Printf("Signing image with key material from '%s'\n", priKeyPath)
```

Comment:
Part of me wonders if we should switch these to sylog.Infof at this point, so they can be `--quiet` or `--silence` 'ed

Arguably they aren't explicit output, but informative messages appropriate for log output rather than direct STDOUT.

Reply:
Though perhaps deferring it til 4.0 is preferable, in case anyone is looking for these on STDOUT.

Reply:
Given that we've changed the content of the console output anyways, it's difficult to imagine a scenario where switching to `INFO` messages would cause additional breakage?

Personally, I'm not sure we can/should provide any guarantee on stability of console output. In cases like this we're adding valuable feedback in the context of a new feature, and that's a good thing! Doesn't mean that one can't choose to parse it, but hopefully it's clear that approach will entail maintenance as Singularity evolves.